src/v/redpanda/admin_server.cc

```diff
@@ -755,9 +755,17 @@ void admin_server::register_partition_routes() {
                 fmt::format("Invalid partition id {}", partition));
           }
 
+          bool is_controller = (ns == "redpanda" && topic == "controller");
```

Comment:
we have `model::controller_ntp` may be worth to use here

Reply:
named type for this in model/namespace.h